onnx_tf/handlers/frontend/bias_add.py

```diff
@@ -1,7 +1,9 @@
+from onnx_tf.common import get_unique_suffix
 from onnx_tf.handlers.frontend_handler import FrontendHandler
 from onnx_tf.handlers.handler import onnx_op
 from onnx_tf.handlers.handler import tf_op
 from .math_mixin import ArithmeticMixin
+import copy
```

Comment:
nit: put build-in library at first and an empty line then third party library then self modules.